test/recipes/30-test_evp_data/evpciph_aes_cts.txt

```diff
@@ -0,0 +1,324 @@
+#
+# Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Original test vectors were taken from https://www.ietf.org/rfc/rfc3962.txt for CS3
```

Comment:
Where do the CS1 and CS2 test vectors come from?

Reply:
It says it down further in the file..
There are none... I just took the CS3 ones and manually changed the output (they are just variants of one another or cbc mode).